### ISSUE TITLE
Include documentation for and test against create-react-native-app

### ISSUE THREAD

**Issue by brentvatne** [FormidableLabs/victory-native#70]
We've had some people running into issues when trying to use victory-native with create-react-native-app and Expo. The most recent one appears to be due to a missing font on Android due to the use of Menlo: https://github.com/FormidableLabs/victory-native/blob/83ec78a88219898e9548653d93cc0e02520717b4/demo/demo.js#L39

You may want to work it into your testing workflow to make sure that victory-native works as expected in CRNA as you release new versions. We actually include it in our version of UIExplorer in Expo, called native-component-list, but we don't always update it to latest.https://github.com/expo/native-component-list/blob/master/screens/SVGScreen.js

**Comment by jevakallio** [comment]
@brentvatne I'm not actively working on victory-native currently, but CRNA is close to my heart so would like to help make this happen. Will take this internally with @boygirl who is the product owner for Victory, and @chrisbolin who's working on it as well.

In terms of Expo/Victory compatibility, we currently require minimum react-native-svg@5.1.5, and the [Expo fork](https://github.com/expo/react-native-svg) is currently at 5.1.2. 

That said, I don't know if
- That fork corresponds to the latest react-native-svg version in a released Expo app
- There is a reason why we need that .5 patch
- How hard would it be for you to bump to higher patch in Expo

**Comment by brentvatne** [comment]
We're on the latest version as per https://github.com/react-native-community/react-native-svg/releases (although we actually pulled in the native side without updating the version for 5.1.3, awkward). Is there even a 5.1.5?

We update on every release of react-native-svg on a monthly basis, we use the latest version available about a week after the newest React Native release.

**Comment by jevakallio** [comment]
Seems like there is 5.1.6 on npm. But I'll wait for Lauren/Chris to provide context, not sure if there is a functional requirement there.

**Comment by chrisbolin** [comment]
@brentvatne thanks for the heads up on the font issue!

**Comment by chrisbolin** [comment]
I made https://github.com/FormidableLabs/victory-native/pull/71 to deal with immediate font issue. I'll keep this issue open as forum to discuss testing against CRNA

**Comment by brentvatne** [comment]
It appears someone ran into an issue with victory-native and CRNA here: https://github.com/react-community/create-react-native-app/issues/159

**Comment by briavicenti** [comment]
Hello! I've been trying to get victory-native to work in my Expo app, but no luck so far. The Expo simulator crashes whenever I try to navigate to a page where I've rendered any of the victory-native elements (ex: a page with just <VictoryBar/>). I need to finish up a project and would love to incorporate some data viz so would greatly appreciate a fix or any suggestions!

**Comment by dikaiosune** [comment]
Released a new CRNA template project. Don't have time just now to test victory-native against it, but there's obviously a lot of interest in using them together so it'd be cool if someone gets the chance to make sure they're still lining up.

**Comment by chrisbolin** [comment]
thanks @dikaiosune! I hope to have time to look into this in the next week.

**Comment by chrisbolin** [comment]
@dikaiosune will i automatically get the new CRNA template if i use `0.0.6`? just checking, as `0.0.6` has been out for a while now

**Comment by chrisbolin** [comment]
I can confirm that `victory-native@0.8.0` plays well with `expo@16.0.0`. 👍👍 

I'll keep this issue open to see if we want to ship a CRNA/expo version of our demo, or at least test with it.

@briavicenti would you mind giving your app a shot with `victory-native@0.8.0`? If that doesn't work, just open up a new victory-native issue. I'll keep this one focused on CRNA testing.

**Comment by chrisbolin** [comment]
@brentvatne @jevakallio @dikaiosune do y'all have any suggestions on what might work best for victory-native CRNA testing / demoing? I don't want the default demo in victory-native to _require_ expo. But we could have another demo? Or what would be the simplest way of testing that the lib works with expo? Do I have to use the expo app, or could this be done in CI?

**Comment by brentvatne** [comment]
@chrisbolin - You'd have to use the Expo app to test it -- it should always remain compatible until the version of react-native-svg changes on either Expo or victory-native. We could probably set something up with [appr](https://github.com/FormidableLabs/appr) to deploy a new version of [this demo app](https://github.com/expo/examples/tree/master/with-victory-native) with the new version of victory-native to make manual testing easier. Anything beyond that would require using something like Appetize.io.

**Comment by chrisbolin** [comment]
thanks @brentvatne! 

i know this is not right venue for this, but I've noticed that CRNA has been failing the first time I use it on projects (including this one) with the error

```
Uncaught Error: 16.0.0 is not a valid SDK version. Options are 15.0.0, 14.0.0, ...
```

**Comment by chrisbolin** [comment]
If i modify `app.json` and use `15.0.0` it works to run `npm start`. Then, and this is where it gets strange, I can kill the process, change it back to `16.0.0`, and restart it - and it works. I've had these exact steps happen to me on two fresh CRNA projects.

**Comment by brentvatne** [comment]
@chrisbolin - you'll need to update the version of Expo on your phone / in your simulator for that

**Comment by chrisbolin** [comment]
closed by #93 - check out https://github.com/FormidableLabs/victory-native-demo
